2001-3000/LIT2866RepCh147.xml

```diff
@@ -57,13 +64,261 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                </bibl>
             </listBibl>
          </div>
-         <div type="edition">
-            <div type="textpart" subtype="incipit" xml:lang="gez">
+         <div type="edition" xml:lang="gez">
+            <div type="textpart">
                <ab>
-                  ሰላም፡ ለዝክረ፡ ስምከ፡ ዘሰሌዳ፡ ሞገስ፡ መጽሐፉ።
+                  <lg type="stanza" n="1">
+                     <l>ሰላም፡ ለዝክረ፡ ስምከ፡ ዘሰሌዳ፡ ሞገስ፡ መጽሐፉ። ዘያሜኒ፡ ኵሎ፡ ወዘያስተሴፉ፡ ፪ማ፡</l>
+                     <l>ጥዑመ፡ ዜና፡ ጊዮርጊስ፡ ለሐሊበ፡ ዕጐልት፡ ሱታፉ፡ ማ፡ ሰላምከ፡ ወረድኤትከ፡ በላዕሌየ፡</l>
+                     <l>ያዕርፉ። አኮ፡ ለምዕር፡ ዳዕሙ፡ ለዝላፉ።</l>
+                  </lg>
+                  <lg type="stanza" n="2">
+                     <l>ሰላም፡ ለድማኅከ፡ በቅብዐ፡ ደመ፡ ስምዕ፡ ዘተሤረየ። አመ፡ ተወክፈ፡ ርእስከ፡ ዘሰፌልያ፡</l>
+                     <l>ሥቃየ፡ ፪ማ፡ ጊዮርጊስ፡ ልዳዊ፡ ቃውመ፡ ነፍስየ፡ ማ፡ እንበለ፡ ይፈጽማ፡ ከናፍርየ፡ ስመ፡</l>
+                     <l>ዚኣከ፡ ስምየ። በጊዜ፡ ምንዳቤ፡ በለኒ፡ ሀለወኩከ፡ ዝየ፡ ።</l>
+                  </lg>
+                  <lg type="stanza" n="3">
+                     <l>ሰላም፡ ለርእስከ፡ ዘዘበጥዎ፡ በመዝረብ። እስከ፡ ይውኅዝ፡ ናላከ፡ ከመ፡ ዘአሥራብ፡ ፪ማ፡</l>
+                     <l>ጊዮርጊስ፡ ልዳዊ፡ ዘቀጰዶቅያ፡ ኮከብ፡ ማ፡ አስተፋጥን፡ ለረዲኦትየ፡ በጊዜ፡ ምንዳቤ፡</l>
+                     <l>ወጻህብ። ለረዲእ፡ ወልዱ፡ ፍቁር፡ ከመ፡ ይጔጕዕ፡ አብ፡ ።</l>
+                  </lg>
+                  <lg type="stanza" n="4">
+                     <l>ሰላም፡ ለገጽከ፡ ዘገለውዎ፡ ዓላውያን፡ በጻሕለ፡ ሐፂን፡ ርሱን፡ ፪ማ፡ ለለጸዋዕኩ፡ ስመከ፡</l>
+                     <l>ውስተ፡ ገጸ፡ ኵሉ፡ መካን፡ ማ፡ ፈረስከ፡ ጽዕድወ፡ ስን፡ እስከ፡ ሶበ፡ ይትነፋሕ፡ ቀርን።</l>
+                     <l>ዕንቋዕ፡ ይብል፡ ጊዮርጊስ፡ ረባን፡ ።</l>
+                  </lg>
+                  <lg type="stanza" n="5">
+                     <l>ሰላም፡ ለቀራንብቲከ፡ ሐረተ፡ ሥቃያት፡ እለ፡ አዝለፋ። መዋዕለ፡ ዓመታት፡ ሰቡዑ፡ እስከነ፡</l>
+                     <l>ሐለፋ፡ ፪ማ፡ ጊዮርጊስ፡ መዋዒ፡ ለትዕግሥት፡ ዘትትዓጸፋ፡ ማ፡ ለነፍስየ፡ ምስለ፡ ነፍስከ፡</l>
+                     <l>ከመ፡ ትረሲ፡ ምዕራፋ፡ ። እትሜነይ፡ ተምኔተ፡ ወእሴፎ፡ ተስፋ፡ ።</l>
+                  </lg>
+                  <lg type="stanza" n="6">
+                     <l>ሰላም፡ ለአዕይንቲከ፡ እለ፡ ኢገገጻ፡ ለርእይ። ትግርምተ፡ ዱድያኖስ፡ መምዕላይ፡ ፪ማ፡</l>
+                     <l>አመ፡ ተገድፈ፡ ሥጋከ፡ ውስተ፡ ክርየተ፡ ግብ፡ ድኁይ፡ ማ፡ በከመ፡ መጠቀከ፡ እደ፡ ፀሐይ፣፡</l>
+                     <l>ዕጓለ፡ ዳግሚት፡ ሰማይ። እምግበ፡ ሙስና፡ ምጥቀኒ፡ ጊዮርጊስ፡ ባሕርይ።</l>
+                  </lg>
+                  <lg type="stanza" n="7">
+                     <l>ሰላም፡ ለአዕዛኒከ፡ ጊዮርጊስ፡ ረቡኒ። አራዊተ፡ ኅለት፡ ኰናኒ፡ ፪ማ፡ እንዘ፡ በአፍአ፡ ሰላመ፡</l>
+                     <l>ይትናገሩኒ፡ ማ፡ ከመ፡ ትስማዕ፡ ወታኅሥሮሙ፡ ለእለ፡ በኅቡዕ፡ ሮዱኒ። ህየንተ፡ ዕዝንየ፡</l>
+                     <l>ዕዝንከ፡ ይኩነኒ።</l>
+                  </lg>
+                  <lg type="stanza" n="8">
+                     <l>ሰላም፡ ለመላትሒከ፡ እለ፡ አውኃዛ፡ ደመ፡ ከመ፡ ሞገድ። አመ፡ ተወቅአ፡ ርእስከ፡</l>
+                     <l>በሰፌልያ፡ ክቡድ፡ ፪ማ፡ ጊዮርጊስ፡ እጹር፡ በውስተ፡ ምክያድ፡ ማ፡ ተማኅፀንኩ፡</l>
+                     <l>በሥቃይከ፡ ዘተወከፍከ፡ በዓውድ። ቤዛ፡ ኃጣውእየ፡ ይኩን፡ ፃማከ፡ ፍድፉድ፡ ።</l>
+                  </lg>
+                  <lg type="stanza" n="9">
+                     <l>ሰላም፡ ለአዕናፊከ፡ እለ፡ ረሰያ፡ ፍጉገ። እምሥዋዐ፡ ጣዖት፡ ርኩስ፡ ጽንሐሐ፡ መዓዛ፡ ዘዓርገ፡</l>
+                     <l>፪ማ፡ ጊዮርጊስ፡ መዋዒ፡ ደራጎነ፡ ቢሩት፡ ፀዋገ፡ ማ፡ ከመ፡ ታዕትት፡ እምአዕናፍየ፡</l>
```

Comment:
There is a single caracter ማ፡ or ፪ማ፡ in every stanza, that I do not understand. Is it an abbreviation, which belongs to the text or is it a transcription error or an editorial sign, that should be deleted from the edition? This ማ፡ or ፪ማ፡ is neither in the manuscript text of EMML 1284, nor in the incipit given by Strelcyn for BLorient12859.